Anixe.Ion/IIonWriter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Anixe.Ion
+{
+    public interface IIonWriter : IDisposable
+    {
+        void WriteSection(string name);
+        void WriteProperty(string name, string value);
+        void WriteProperty(string name, int value);
+        void WriteProperty(string name, double value);
+        void WriteProperty(string name, float value);
```

Comment:
class IonWriter also implements `void WriteProperty(string name, bool value);` and `void WriteProperty(string name, char value);`